lib/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import classnames from 'classnames';
+import { useStyles } from 'sku/treat';
+import { Theme } from 'treat/theme';
+import { Box } from '../Box/Box';
+import { useBackground } from '../Box/BackgroundContext';
+import { UseBoxStylesProps } from '../Box/useBoxStyles';
+import * as styleRefs from './Loader.treat';
+
+const indicators = [...Array(3)];
+
+interface LoaderProps {
+  size?: keyof Theme['typography']['text'];
```

Comment:
I don't think we should be importing `Theme` directly from `treat`. Should we model this as `size?: keyof typeof styleRefs.size`?

Reply:
Yep now we can, that was a relic of a refactor. FWIW though i was followed this pattern from [`useBoxStyles`](https://github.com/seek-oss/braid-design-system/blob/25efb2c32cfae990b2775e7a1a1f7cc6b77f538a/lib/components/Box/useBoxStyles.ts#L12)